zellij-tile/src/shim.rs

```diff
@@ -37,6 +45,14 @@ pub fn open_file(path: &Path) {
 pub fn set_timeout(secs: f64) {
     unsafe { host_set_timeout(secs) };
 }
+pub fn exec_cmd(cmd: &[String]) -> Result<(), PermissionDenied> {
+    if !get_exec_permission() {
```

Comment:
No need to have this check here once the check has been moved inside of `host_exec_cmd()` in `wasm_vm.rs`!